lib/state/action/actions.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex_async_redux/state/app_state.dart';
+
+//Loading action
```

Comment:
this is a comment not a documentation, it should be ///. Also the message doesn't tell anything what this comment is for 😝